docs/user_guide/geometric_representations.mdx

```diff
@@ -336,6 +379,10 @@ Keep in mind that while those constructors will fail if the topology of the conv
 
 </div>
 
+{/* Would `SimdCompositeShape` be relevant here? */}
+
+{/*
+
 # Composite shapes
```

Comment:
I'm not sure how to document this part ; I feel like this would be now referring to `SimdCompositeShape`, but not sure, I'd appreciate help :)

Reply:
yes